src/elements/content-sidebar/AddTaskMenu.js

```diff
@@ -17,11 +17,12 @@ import './AddTaskMenu.scss';
 type Props = {
     isDisabled: boolean,
     onMenuItemClick: (taskType: TaskType) => void,
+    setAddTaskButtonRef?: (element: HTMLButtonElement) => void,
 };
 
 const AddTaskMenu = (props: Props) => (
     <DropdownMenu constrainToScrollParent isRightAligned>
-        <Button isDisabled={props.isDisabled} type="button">
+        <Button isDisabled={props.isDisabled} type="button" setRef={props.setAddTaskButtonRef}>
```

Comment:
You can destruct the `props` in function parameter to extract `isDisabled` and `setAddTaskButtonRef`.

Reply:
We can and should, but it looks like the whole file is written this way. Maybe as a cleanup PR?

Reply:
What's our rationale behind enforcing destructuring?

Reply:
It's defined in our eslint config, I think from airbnb's, as [react/destructuring-assignment](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md). This project it has it disabled due to there being so many instances of noncompliance..